#goal definition
geometry_msgs/PointStamped fixation_point
---
#result definition
bool state_reached
---
#feedback
float64[] joint_states
